sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -1596,3 +1608,435 @@ def _more_tags(self):
                 'zero sample_weight is not equivalent to removing samples',
             }
         }
+
+
+class SGDOneClassSVM(BaseSGD, OutlierMixin):
+    """Solves linear One-Class SVM using Stochastic Gradient Descent.
+
```

Comment:
We could add a brief mention to Nystroem here, to extend this class to non-linear kernels.

No need to go in great details, as it is already in the user guide and in both examples, but a brief mention here could still be useful.